### ISSUE TITLE
Performance on Resnet101 network

### ISSUE THREAD

**Issue by twtygqyy** [chenyuntc/simple-faster-rcnn-pytorch#32]
Hi, I've implemented the resnet101 structure on the top of vgg16 network, while the mAP on VOC datasets could only reach 0.62 after 20 epochs.
Do you have any idea what the problem would be? You can find the code [here](https://github.com/twtygqyy/simple-faster-rcnn-pytorch/blob/master/model/faster_rcnn_resnet101.py). Thank you.

**Comment by chenyuntc** [comment]
Resnet101 is difficult to train. see [here](https://github.com/ruotianluo/pytorch-faster-rcnn/blob/master/README.md)for more info.

>For Resnets, we fix the first block (total 4) when fine-tuning the network, and only use crop_and_resize to resize the RoIs (7x7) without max-pool (which Xinlei finds useless especially for COCO). The final feature maps are average-pooled for classification and regression. All batch normalization parameters are fixed. Learning rate for biases is not doubled.

**Comment by twtygqyy** [comment]
@chenyuntc Thanks, I also fixed the weights for top layers, but the result didn't improve. As you mentioned, it might be the reason of BN and biases. I'll have another try.

**Comment by chenyuntc** [comment]
Actually, I only tried VGG16.

**Comment by blateyang** [comment]
I recently  also want to implement resnet structure based on this project. And I found your @twtygqyy codes are very helpful to me. But I have a question about batch normalization. Why we need to fix batch normalization parameters here?

**Comment by stickOverCarrot** [comment]
@blateyang BN only work when batch_size>1 and only work well when batch_size>=16.You can see this paper https://arxiv.org/abs/2002.05712.However，@chenyuntc code only surport batch_size==1

**Comment by blateyang** [comment]
Thanks for your reply!

stickOverCarrot <notifications@github.com> 于2020年10月17日周六 下午7:48写道：
